mls-rs-uniffi/tests/simple_scenario_sync.py

```diff
@@ -7,35 +7,35 @@ def __init__(self, id: "int", data: "bytes"):
         self.data = data
 
 class GroupStateData:
-    def __init__(self, state: "bytes"):
+    def __init__(self, state: bytes):
         self.state = state
-        self.epoch_data = []
+        self.epoch_data: list[EpochRecord] = []
 
 class PythonGroupStateStorage(GroupStateStorage):
     def __init__(self):
-        self.groups = {}
+        self.groups: dict[str, GroupStateData] = {}
 
-    def state(self, group_id: "bytes"):
+    def state(self, group_id: bytes):
         group = self.groups.get(group_id.hex())
 
         if group == None:
             return None 
 
         group.state
 
-    def epoch(self, group_id: "bytes",epoch_id: "int"):
+    def epoch(self, group_id: bytes, epoch_id: int):
         group = self.groups[group_id.hex()]
 
         if group == None:
             return None
-        
+
         for epoch in group.epoch_data:
             if epoch.id == epoch_id:
                 return epoch
 
         return None
 
-    def write(self, state: "GroupState",epoch_inserts: "typing.List[EpochRecord]",epoch_updates: "typing.List[EpochRecord]"):
+    def write(self, state: "GroupState", epoch_inserts: list[EpochRecord], epoch_updates: list[EpochRecord]):
```

Comment:
`GroupState` is not actually defined yet — I'll clean this up eventually by importing it from the generated Python code.